libs/api/domains/form-system/src/lib/company/companyRegistry.resolver.ts

```
@@ -0,0 +1,51 @@
+import { CodeOwner } from '@island.is/nest/core'
+import { CodeOwners } from '@island.is/shared/constants'
+import {
+  UseGuards,
+  BadRequestException,
+  ForbiddenException,
+} from '@nestjs/common'
+import { Args, Query, Resolver } from '@nestjs/graphql'
+import { CompanyRegistryClientService } from '@island.is/clients/rsk/company-registry'
+import { CompanyExtendedInfo } from '../../models/company.model'
+
+type CurrentUser = {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'CurrentUser'. Consider to delete it or rename it.

'CurrentUser' is defined here:


(lint/suspicious/noRedeclare)